sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -1084,6 +1117,20 @@ export interface ServiceBusDeadletterMessagesAvailableWithNoListenersEventData {
 // @public
 export type StampKind = string;
 
+// @public
+export interface StorageAsyncOperationInitiatedEventData {
```

Comment:
same thought - if they have the same shape, I'd love to declare it once and alias it